packages/flutter/test/widgets/interactive_viewer_test.dart

```diff
@@ -295,7 +295,7 @@ void main() {
           home: Scaffold(
             body: Center(
               child: InteractiveViewer(
-                alignPanAxis: true,
+                alignPanAxis: PanAxis.aligned,
```

Comment:
Can you update the name of this test and the one below?  It says alignPanAxis but should probably say "PanAxis.aligned" now.